Dockerfile

```diff
@@ -0,0 +1,18 @@
+#Using the official ubuntu OS as a Base Image
+FROM ubuntu:16.04
+
+USER root
+
+#Configuration
+RUN pip install -r requirements/base.txt
+RUN paver install_pages
+
+#Setting up environment variables. Change the value according to the value of the ENV_VAR
+
```

Comment:
We need a few more environment variables here. You might want to make use of local_env.sh